From: Jens Axboe <axboe@kernel.dk>
Subject: task_work: add kerneldoc annotation for 'data' argument
Date: Mon, 11 Sep 2023 14:24:06 -0600

A previous commit changed the arguments to task_work_cancel_match(), but
didn't document all of them.

Link: https://lkml.kernel.org/r/93938bff-baa3-4091-85f5-784aae297a07@kernel.dk
Fixes: c7aab1a7c52b ("task_work: add helper for more targeted task_work canceling")
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202309120307.zis3yQGe-lkp@intel.com/
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/task_work.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/task_work.c~task_work-add-kerneldoc-annotation-for-data-argument
+++ a/kernel/task_work.c
@@ -78,6 +78,7 @@ int task_work_add(struct task_struct *ta
  * task_work_cancel_match - cancel a pending work added by task_work_add()
  * @task: the task which should execute the work
  * @match: match function to call
+ * @data: data to be passed in to match function
  *
  * RETURNS:
  * The found work or NULL if not found.
_
